### DISCUSSION TITLE
Radioking tx18s compatibility?

### DISCUSSION THREAD

**Discussion by RipperGOT** [EdgeTX/edgetx#680] [General]
Hi guys! What's happens with this radio? Have support? Screen works?
In theory it's a cheaper Jumper T18.

**Comment by raphaelcoeffic** [comment]
Yes, everything is fine so far, just uses the T18 target (should have touch screen as well).

**Reply by RipperGOT** [reply]
> Yes, everything is fine so far, just uses the T18 target (should have touch screen as well).

Thanks for your replie!!!!!

I am looking forward to this radio and try this new software that seems very good!
